CONTRIBUTING.md

```diff
@@ -127,6 +127,25 @@ reference GitHub issues that this commit **Closes**. In order to maintain a refe
 `fixes #<issue_number>` if it closes a related issue or `issue #<issue_number>`
 if it's a partial fix.
 
+## What should you do once your PR is merged?
+
+Once your PR is merged, it's recommended to pull new changes from the `main` (which will also include the one that you made).
+So, for that purpose specify a new remote `upstream` repository which will be pointing to the actual [mozilla/fxa](https://github.com/mozilla/fxa) repo.
+
+git remote add upstream https://github.com/mozilla/fxa.git
```

Comment:
Can we add backticks around this too?

Reply:
Yeah sure! If we're keeping this change I will commit the required changes.